src/app/Topology/Shared/utils.tsx

```diff
@@ -131,16 +130,15 @@ export const isTargetNodeFiltered = ({ target }: TargetNode, filters?: TopologyF
     matched = matched && target.jvmId !== undefined && filters.JvmId.includes(target.jvmId);
   }
   if (filters.Label && filters.Label.length) {
-    matched =
-      matched && Object.entries(target.labels || {}).filter(([k, v]) => filters.Label.includes(`${k}=${v}`)).length > 0;
+    matched = matched && target.labels.map(keyValueToString).some((v) => filters.Label.includes(v));
```

Comment:
```suggestion
    matched = matched && target.labels.some((kv) => filters.Label.includes(keyValueToString(kv)));
```

I guess it's just cleaner like line 139 below.